website/docs/Conditions.md

```diff
@@ -0,0 +1,166 @@
+---
+slug: Conditions.md
+sidebar_position: 20
+---
+
+# Conditions
+Several types of access conditions can be defined:
+- EVM - on chain state, eg NFT ownership, ETH balance, tx status, contract function call
+- RPC - ethereum RPC calls as defined in the [Official API](https://ethereum.org/en/developers/docs/apis/json-rpc/#json-rpc-methods)
```

Comment:
There is a current allow list of methods that limits the calls that can be made, no -https://github.com/nucypher/nucypher/blob/tdec/nucypher/policy/conditions/evm.py#L96?

cc @KPrasch

Reply:
My 2 cents:
- At this moment the EVM condition supports only ERC20 contract (with ERC721 arriving shortly); no testing with custom ABIs has been done
- `nucypher-ts` supports `eth_balance` and `balanceOf` methods (tested).